controllers/toplevel/index.js

```diff
@@ -99,104 +95,60 @@ const newHomepage = (req, res) => {
     }
 };
 
-// @TODO cache this page as it's very slow to return
-// main issue is the static assets (which cloudfront doesn't cache)
 const oldHomepage = (req, res) => {
-    // don't cache this page!
-    res.cacheControl = { maxAge: 0 };
+    return proxyLegacy.proxyLegacyPage(req, res);
+};
 
-    // work out if we need to serve english/welsh page
-    let localePath = req.i18n.getLocale() === 'cy' ? config.get('i18n.urlPrefix.cy') : '';
-
-    return rp({
-        url: legacyUrl + localePath,
-        strictSSL: false,
-        jar: true,
-        resolveWithFullResponse: true
-    })
-        .then(response => {
-            let body = response.body;
-            // convert all links in the document to be absolute
-            // (only really useful on non-prod envs)
-            body = absolution(body, 'https://www.biglotteryfund.org.uk');
-
-            // fix meta tags in HTML which use the wrong CNAME
-            body = body.replace(/wwwlegacy/g, 'www');
-
-            // parse the DOM
-            const dom = new JSDOM(body);
-
-            const form = dom.window.document.getElementById('form1');
-            if (form) {
-                const newAction = form.getAttribute('action').replace('https://www.biglotteryfund.org.uk/', '/');
-                form.setAttribute('action', newAction);
-            }
+// serve the legacy site funding finder (via proxy)
+router.get('/funding/funding-finder', (req, res) => {
+    // parse the string (eg. "£10,000" => 10000, "£1 million" => 1000000 etc)
+    let parseValueFromString = str => {
```

Comment:
@mattandrews If we're going to add unit tests for any of this, this `parseValueFromString` function is probably a good candidate.

Reply:
Have a look at the newest commits!